src/main/java/org/folio/services/validator/InvoiceValidator.java

```diff
@@ -121,7 +121,14 @@ private void validateInvoiceLineFundDistributions(List<InvoiceLine> invoiceLines
       if (CollectionUtils.isEmpty(line.getFundDistributions())) {
         throw new HttpException(400, FUND_DISTRIBUTIONS_NOT_PRESENT);
       }
-      validateFundDistributions(line.getTotal(), line.getFundDistributions());
+
+      try {
+        validateFundDistributions(line.getTotal(), line.getFundDistributions());
+      } catch (HttpException e) {
+        throw new HttpException(422,INCORRECT_FUND_DISTRIBUTION_TOTAL, Lists.newArrayList(new Parameter()
+          .withKey(INVOICE_LINE_NUMBER)
+          .withValue(line.getInvoiceLineNumber())));
```

Comment:
And amount param we don't need here?

Reply:
It will display when navigate to POL. During yesterday's refinement we really don't need it

Reply:
FInal message displayed on UI would be like this: Fund distribution total must be distributed by 100 percent in the invoice line number {{invoiceLineNumber}}